dataflow-website/recipes/scaling/scdf-alert-webhook/README.md

```diff
@@ -0,0 +1,20 @@
+
+# Auto-scaling adapter for Streaming Data Pipelines
+
+The `AlertWebHookApplication` is a [Prometheus Alertmanager Webhook Receiver](https://github.com/prometheus/alertmanager) 
+that listens for pre-configured Prometheus alerts and leverages the SCDF Scale API to scale out or in a preconfigured 
+stream application.
+
+Use the `scdf.alert.webhook.scaleOutAlertName` and  `scdf.alert.webhook.scaleInAlertName` to configure the names of the scale-out and scale-in alert names.
+
+On scale-out alert, the `AlertWebHookApplication` increases the number of application (defined by `scdf.alert.webhook.scaleApplicationName` or `application_name` alert label) instances  to the `scdf.alert.webhook.scaleOutFactor` count.
```

Comment:
Ah, this answers my question about why `transform` or the app name is listed as env-var. Folks can override to change it I suppose.

Reply:
indeed. My ideal was to provide a bit more flexible app that can be reused with different pipelines and apps